tests/sentry/web/frontend/test_react_page.py

```diff
@@ -309,11 +308,12 @@ def test_customer_domain_org_pending_deletion(self):
         self.login_as(self.user)
 
         with self.feature({"organizations:customer-domains": [org.slug]}):
-            response = self.client.get(
-                "/issues/",
-                SERVER_NAME=f"{org.slug}.testserver",
-                follow=True,
-            )
+            with assume_test_silo_mode(SiloMode.REGION):
+                response = self.client.get(
```

Comment:
We may want to add `outbox_runner` and `receivers_raise_on_send` context handlers here to validate and test the actual signal being executed.